osu.Game/OsuGame.cs

```diff
@@ -1040,7 +1040,7 @@ private void forwardGeneralLogsToNotifications()
 
             Logger.NewEntry += entry =>
             {
-                if (entry.Level < LogLevel.Important || entry.Target > LoggingTarget.Database) return;
+                if (entry.Level < LogLevel.Important || entry.Target is null or > LoggingTarget.Database) return;
```

Comment:
We don't use such syntax in comparisons, it reads bad:
```suggestion
                if (entry.Level < LogLevel.Important || entry.Target > LoggingTarget.Database || entry.Target == null) return;
```